Vienna Tests/models/ArticleTests.m

```diff
@@ -348,4 +349,18 @@ - (void)testExpandArticleEnclosureFileName
     XCTAssertEqualObjects(expandedString, expectedString);
 }
 
+- (void)testURLIDNinArticleView
+{
+    ArticleView * view = [[ArticleView alloc] initWithFrame:NSMakeRect(0, 0, 100, 100)];
+
+    self.article.link = @"http://ουτοπία.δπθ.gr/نجيب_محفوظ/";
+    NSString * htmlText = [view articleTextFromArray:@[self.article]];
+    [view setHTML:htmlText];
+
+    self.article.link = @"http://xn--kxae4bafwg.xn--pxaix.gr/%D9%86%D8%AC%D9%8A%D8%A8_%D9%85%D8%AD%D9%81%D9%88%D8%B8/";
+    NSString * htmlText2 = [view articleTextFromArray:@[self.article]];
+
+    XCTAssertEqualObjects(htmlText, htmlText2);
```

Comment:
It would be helpful if we could use meaningful variable names here, something like `expectedURL` and `URLfromIDN`.